lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -697,6 +699,20 @@ newtype ApiWalletPassphrase = ApiWalletPassphrase
     } deriving (Eq, Generic, Show)
       deriving anyclass NFData
 
+newtype ApiWalletUtxoSnapshot = ApiWalletUtxoSnapshot
```

Comment:
I think it's possible to avoid the extra type by just using `ApiT [ApiWalletUtxoSnapshotEntry]`.